custom_components/frigate/media_source.py

```diff
@@ -1277,17 +1274,10 @@ def _browse_recordings(
 
         for recording in recordings:
             title = self._generate_recording_title(identifier, recording)
-            if not title:
```

Comment:
I haven't been able to figure out how to actually get a ValueError to result in the title being None in this context because all the parts of the identifier are validated upstream when parsing the identifier. Also, the folder contents under the camera are longer parsed since recordings are now hour long VOD playlists.

Reply:
I think this can still happen with a syntactically valid but semantically invalid date, i.e. [this test](https://github.com/blakeblackshear/frigate-hass-integration/blob/master/tests/test_media_source.py#L848) I think is triggering the earlier catch of the ValueError. The upstream validation is checking they are valid numbers, not necessarily valid dates when assembled.

If I'm missing something and this is really just dead code, then lets just remove the exception handling of ValueError in `_generate_recording_title` rather than catching it but not handling the `None`. That'll cause the ValueError to propagate all the way to the top which I think is appropriate if we think this cannot ever happen.

Reply:
I can hit the ValueError in the `test_async_browse_media_recordings_root` test, but I can't find a way to hit it when it's being called from the `_browse_recordings` method where I removed this check. The exception handling isn't dead code, but it doesn't seem possible to have title be None in this particular method. I couldn't get back to 100% coverage without removing this check.